nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -275,15 +281,17 @@ def _prepare_subgraph(self, node: NNCFNode, model: TModel, nncf_graph: NNCFGraph
         :param subgraph_data: A dictionary with the layers for the graph building.
         :return: Backend-specific subgraph extracted from the model.
         """
-        input_node_names, output_node_names = subgraph_data["input_node_names"], subgraph_data["output_node_names"]
+        input_node_names, output_node_names = (
+            subgraph_data["subgraph_input_names"],
+            subgraph_data["subgraph_output_names"],
+        )
         extracted_model = self.extract_model(model, input_node_names, output_node_names)
 
         transformation_layout = TransformationLayout()
         model_transformer = ModelTransformerFactory.create(extracted_model)
-        _, output_port_id = self._backend_entity.get_activation_port_ids_for_bias_node(node)
-        statistic_point = self._backend_entity.target_point(
-            TargetType.POST_LAYER_OPERATION, node.node_name, output_port_id
-        )
+
+        # For layes with weights, there is only one output port - 0.
+        statistic_point = self._backend_entity.target_point(TargetType.POST_LAYER_OPERATION, node.node_name, port_id=0)
```

Comment:
It's also 0 for some nodes without weights. Which variable do we need in this case?

Reply:
It's looks like magic number, that should be replaced to symbolic constant.

> It's also 0 for some nodes without weights. Which variable do we need in this case?

But in this place it's used only for weighted? It can be names `OUTPUT_PORT_OF_NODE`

Reply:
Added